src/pages/dashboard/_CDCComparisonContainer.js

```diff
@@ -82,6 +87,10 @@ export default function CDCComparisonContainer() {
           />
         </div>
         <div style={{ flexGrow: 1, width: '50%' }}>
+          <div>
+            <h4 style={{ marginBottom: '0.02rem' }}>Covid tracking project</h4>
```

Comment:
Avoid inline styles, and also we now have logic upstream that switches this title depending on the viewport to avoid line breaks.

https://github.com/COVID19Tracking/website/blob/a3c01a98245792e168daffc4c5ded5ac38b554a0/src/pages/dashboard/_CDCComparisonContainer.js#L151